Resources/Public/JavaScript/Modules/Edit.js

```diff
@@ -44,15 +50,43 @@ function submitFormHandler(event) {
 	return false;
 }
 
+function disableElement(element) {
+	element.classList.add('disabled');
+
+	element.addEventListener('click', function (event) {
+		event.preventDefault();
+		return false;
+	});
+
+	var buttonTitleElement = element.querySelector('span');
+	if (buttonTitleElement != null) {
+		buttonTitleElement.title = 'Disabled due to … ' + buttonTitleElement.title;
```

Comment:
I need to find a proper text.
Also this text should not be hard coded but be fetched from localization file.